src/js/charts/line.js

```diff
@@ -110,6 +112,14 @@
           .attr('d', plot.area(args.data[which_line]))
           .attr('clip-path', 'url(#mg-plot-window-' + mg_target_ref(args.target) + ')');
       } else { // otherwise, add the area
+        svg.append('path')
+          .classed('mg-main-area', true)
+          .classed('mg-area' + line_id, true)
+          .classed('mg-area-color')
+          .attr('d', plot.area(args.data[which_line]))
+          .attr('fill', 'black')
+          .attr('clip-path', 'url(#mg-plot-window-' + mg_target_ref(args.target) + ')');
```

Comment:
This is the one part of the code that I'm unsure about (I think it might be what's causing the unit tests to fail). It doesn't seem correct to basically copypaste this rather large block of code twice. If you need to add a class to the path, can you not do that by appending a classed statement to the original set of calls below?

Reply:
Yeah that's exact place where the problem was occurring. Removing that code passed all the tests.
I'll try and figure out a way to correct that.

Reply:
Yep, 100% sure that `fill black ` is causing the error. What can be substituted in that place ?

Reply:
Offhand, I am not sure and don't have time to test right now. Maybe it's just the tests that need to be updated? I would experiment a bit. If you're still stuck after trying some stuff let me know and I'll see if I can help.

Reply:
The problem has been solved though !

The classes being pushed in this 
https://github.com/shikhar-scs/metrics-graphics/blob/c063971799a5d2eaaf35c7ad7660ba793a9abe5e/src/js/charts/line.js#L4-L9

and

https://github.com/shikhar-scs/metrics-graphics/blob/c063971799a5d2eaaf35c7ad7660ba793a9abe5e/src/js/charts/line.js#L115-L121


already have the `fill` attribute set to black , i.e. look at `mg-hover-line-color` and `mg-area-color`. So, basically we dont need to overwrite them further . 


I'm pushing the final (hopefully) code. Please review :+1: